src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc

```diff
@@ -744,234 +744,80 @@ Defines the display element from the selected report the page block will use whe
-    "alias": "compose-configuration-page-block-reference-added-navigation-justify-none",
-    "w": 1920,
-    "h": 1080
-  },
-  "view": {
-    "x": 321,
-    "y": 0,
-    "h": 400,
-    "w": 1599
-  },
-  "annotations": []
-}
-----
-
+Allows you to create a new tab.
+[NOTE]
+====
+Tabbing a block removes it from the main page.
+====
 
-| [#page-block-navigation-alignment]#<<page-block-navigation-alignment,Alignment>>#
+| [#page-block-tabs-title]#<<page-block-tabs-title,Title>>#
 |
-Defines the text alignment of each navigation item
-
-.Supported alignments:
-* Left +
+Determines the title of the tab. If not set, the title of the page block is used, and if that is unset, a generic name is used which is based on the kind of the page block.
 
-[annotation,role="data-zoomable"]
-----
-{
-  "image": {
-    "rel": "compose-configuration/page-block-reference/added-navigation-left.png",
-    "alias": "compose-configuration-page-block-reference-added-navigation-left",
-    "w": 1920,
-    "h": 1080
-  },
-  "view": {
-    "x": 321,
-    "y": 0,
-    "h": 400,
-    "w": 1599
-  },
-  "annotations": []
-}
-----
-
-* Center +
-
-[annotation,role="data-zoomable"]
-----
-{
-  "image": {
-    "rel": "compose-configuration/page-block-reference/added-navigation-center.png",
-    "alias": "compose-configuration-page-block-reference-added-navigation-center",
-    "w": 1920,
-    "h": 1080
-  },
-  "view": {
-    "x": 321,
-    "y": 0,
-    "h": 400,
-    "w": 1599
-  },
-  "annotations": []
-}
-----
-
-* Right +
-
-[annotation,role="data-zoomable"]
-----
-{
-  "image": {
-    "rel": "compose-configuration/page-block-reference/added-navigation-right.png",
-    "alias": "compose-configuration-page-block-reference-added-navigation-right",
-    "w": 1920,
-    "h": 1080
-  },
-  "view": {
-    "x": 321,
-    "y": 0,
-    "h": 400,
-    "w": 1599
-  },
-  "annotations": []
-}
-----
+| [#page-block-tabs-edit]#<<page-block-tabs-edit,Edit>>#
+|
+Allows you to access the configurator of a tabbed block.
 
-| [#page-block-navigation-item-type]#<<page-block-navigation-item-type,Navigation item type>>#
+| [#page-block-tabs-plus]#<<page-block-tabs-plus,Add Block>>#
 |
-Defines the functionality of a navigation item
+Allows you to add a new block that is immediately tabbed. Access this configuration by clicking on the btn:[+] button.
 
-.Supported navigation item types:
-* Text: This type displays a navigation item as just test with no extra functionalities.
-* URL: This type of navigation item has a URL option. When clicked, it will redirect to the corresponding URL.
-* Compose page: This type of navigation item has a Compose page option. When clicked, it will redirect to the corresponding Compose page.
-* Dropdown: This type of navigation item has a dropdown option. When clicked, it will display a dropdown with the corresponding dropdown items.
+| [#page-block-tabs-delete]#<<page-block-tabs-delete,Delete>>#
```

Comment:
The configurator options down here are page block specific, this is a generic thing (deleting page blocks) so it doesn't belong here

Reply:
Added it there because the [Note] is important